src/esp/scene/SceneNode.h

```diff
@@ -88,6 +118,15 @@ class SceneNode : public MagnumObject,
   //! this node is the root
   const Magnum::Range3D& getCumulativeBB() const { return cumulativeBB_; };
 
+  // Return SensorSuite containing references to Sensors this SceneNode holds
+  esp::sensor::SensorSuite& getNodeSensorSuite() { return *nodeSensorSuite_; }
```

Comment:
We should also provide another 2 wrapper functions for convenience:
`getNodeSensors()` which will return `nodeSensorSuite_->getSensors()`
and 
`getSubtreeSensor()` ...